polaris-react/src/components/Tag/Tag.scss

```diff
@@ -27,118 +28,77 @@ $button-size: 20px;
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     @include unstyled-button;
     cursor: pointer;
-    padding: var(--p-space-1) var(--p-space-2);
+    padding: 0 calc(var(--p-space-1) + var(--p-space-05));
     background-color: var(--p-color-bg-strong);
     outline: var(--p-border-width-1) solid transparent;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    font-size: 13px;
-    line-height: var(--p-font-line-height-2);
+    font-size: var(--p-font-size-80-experimental);
```

Comment:
Updated `font-size: 13px` in this file to use the token so that we could clean up stylelint comments.